pulumi/grapl/__main__.py

```diff
@@ -420,17 +426,6 @@ def main() -> None:
             nomad_agent_security_group_id=nomad_agent_security_group_id,
         )
 
-        pulumi.export("plugin-registry-db-hostname", plugin_registry_postgres.host())
```

Comment:
These may need to be added in later for the pulumi integration tests project. But since its not being used at the moment, makes sense to remove it